tests/server/wildfly/src/test/java/org/hawkular/apm/tests/wildfly/AnalyticsServiceRESTTest.java

```diff
@@ -328,15 +329,12 @@ public void testGetCompletionCountWithPropertyFilter() {
         trace1.setStartTime(System.currentTimeMillis() - 4000); // Within last hour
         Consumer c1 = new Consumer();
         c1.setUri("testuri");
+        c1.getProperties().add(new Property("prop1", "2.5", PropertyType.Number));
+        c1.getProperties().add(new Property("prop2", "hello"));
         trace1.getNodes().add(c1);
-        trace1.getProperties().add(new Property("prop1", "2.5", PropertyType.Number));
-        trace1.getProperties().add(new Property("prop2", "hello"));
-
-        List<Trace> traces = new ArrayList<Trace>();
-        traces.add(trace1);
 
         try {
-            publisher.publish(null, traces);
+            publisher.publish(null, Arrays.asList(trace1));
```

Comment:
Perhaps it has the same effect, but I have the feeling that a `Collections.singletonList()` would be more appropriate. In any case, feel free to ignore this comment, as this is test code :)